apps/web/app/(app)/premium/Pricing.tsx

```
@@ -155,18 +155,24 @@ export function Pricing(props: { header?: React.ReactNode }) {
-              : "/login?next=/premium";
+            function getHref(): string {
+              if (!user) return "/login?next=/premium";
+
+              if (isCurrentPlan) return "#";
+
+              if (tier.ctaLink) return tier.ctaLink;
+
+              return buildLemonUrl(
+                attachUserInfo(tier.href[frequency.value], {
+                  id: user.id,
+                  email: user.email!,
```

Biome

[error] Forbidden non-null assertion.



(lint/style/noNonNullAssertion)